src/critic_tests/run_checks_tests.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+from critic.models import UptimeMonitor, MonitorState
+import boto3
+import time
+from uuid import uuid4
+from datetime import datetime, timedelta
+
+
+@pytest.fixture
+def get_uptime_monitor():
+    return UptimeMonitor(project_id=uuid4(), 
+                        slug="test-slug", 
+                        url="google.com",
+                        timeout_secs=3,
+                         )
+
+def test_run_checks():
+    client = boto3.client('dynamodb', region_name='us-east-2')
+
+    #run test function
+
+    # check if the function worked by checking ddb for log entries!
+
+
+
+
+#not sure where to put this for now
+def run_checks(monitor: UptimeMonitor):
+    import requests
+    #ping website
+    start = time.perf_counter()
+    
+    try:
+        response = requests.head(monitor.url, timeout=monitor.timeout_secs) 
+        finished = time.perf_counter()
+        time_to_ping = start - finished
+    except requests.RequestException:
+        response = None #what do we do for None? Add a failed state?
```

Comment:
Eventually we should use Sentry for unanticipated errors: https://github.com/calebsyring/critic/issues/31

We also need a mechanism for retrying tasks, but that will probably be handled outside the task itself.

For anticipated errors, like the request timing out, we should handle those as a failure. Monitor should be set to down, etc.